scripts/init-repo/branchpermissions.ps1

```diff
@@ -0,0 +1,42 @@
+
+function Invoke-InitRepoSectionBranchPermissions
```

Comment:
Some descriptions in these functions would be fine. I would be good to see on a first look, what it does and what it setups

Reply:
good idea, will do. thanks.

Reply:
described